GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -844,4 +845,74 @@ void ExporterHelper::AddDeprecatedFontFilesToFontResources(
   // end of compatibility code
 }
 
+const std::array<int, 20> IOS_ICONS_SIZES = {
+    180, 60,  120, 76, 152, 40, 80, 57,  114, 72,
+    144, 167, 29,  58, 87,  50, 20, 100, 167, 1024,
+};
+const std::array<int, 6> ANDROID_ICONS_SIZES = {36, 48, 72, 96, 144, 192};
+
+const gd::String ExporterHelper::GenerateWebManifest(
+    const gd::Project &project) {
+  const gd::String &orientation = project.GetOrientation();
+  gd::String icons = "[";
+
+  {
+    std::map<int, gd::String> resourcesForSizes;
+    const auto FileNameForIcon = [&project](const gd::String &platform,
```

Comment:
Uber nitpicking:

this is a local variable, so camelCase + this a function, so always start with a verb

```suggestion
    const auto getFileNameForIcon = [&project](const gd::String &platform,
```